test/parallel/test-bootstrap-modules.js

```diff
@@ -9,7 +9,7 @@ const common = require('../common');
 const assert = require('assert');
 
 const isMainThread = common.isMainThread;
-const kMaxModuleCount = isMainThread ? 58 : 80;
+const kMaxModuleCount = isMainThread ? 59 : 81;
```

Comment:
This is incremented because previously we use `getInternalBinding` to load `native_module` which bypasses the caching and `process.moduleLoadList` bookkeeping. This patch uses `internalBinding` instead so it is now being counted into the `process.moduleLoadList`